zephyr/wrapper.c

```diff
@@ -677,6 +687,11 @@ int arch_cpu_enable_core(int id)
 {
 	pm_runtime_get(PM_RUNTIME_DSP, PWRD_BY_TPLG | id);
```

Comment:
it is odd that we never `pm_runtime_put()` when the core is disabled...

Reply:
Ack @ujfalusi  . This is basicly just missing plumbing. Also a bit surpising is that pm_runtime_get/put is not a ref-counting based implementation, so _put directly powers the core down. Otherwise we could have taken an extra reference and hid the problem in a nice way. Now we simple cannot call put() as it will currently lead to crashes and havoc.